public/src/components/common/list_view.jsx

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
 import { Table } from 'react-bootstrap';
 
-import AdminTableRow from './admin_table_row';
-import AdminDashboardButtonGroup from './admin_nav_btn_group';
-import AdminBanner from './admin_welcome_banner';
-import CreateNewButton from './admin_createNew_btn';
+import TableRow from './table_row';
+import DashboardButtonGroup from './nav_btn_group';
+import Banner from './welcome_banner';
+import CreateNewButton from './createNew_btn';
```

Comment:
I didn't catch this before, I think we should stay consistent with file names and change this to create_new_button

Reply:
I agree. This is an artifact of the way the file name came to me, I only took "admin_" off the front of the component file names.